ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -21,6 +21,8 @@ let prototyped_of_field = function
       Some "22.20.0"
   | "VM", "actions__after_softreboot" ->
       Some "23.1.0"
+  | "pool", "last_update_sync" ->
+      Some "23.3.0-next"
```

Comment:
```
      Some "23.5.0-next"
```

Reply:
These new fields are to be added into feature branch firstly. Maybe the lifecycle of new added fields could be added just before merging to master?

Reply:
that sounds good, you'll have to remember to to commit them before that point

Reply:
I think before the code is finally merged from the feature branch to master, a local make will re-update datamodel_lifecycle.ml to the latest version, will add that change then.